apps/passport-client/components/screens/FrogScreens/Button.tsx

```diff
@@ -16,7 +16,7 @@ export function ActionButton({
    * The action to perform when the button is clicked. This should return a
    * promise that resolves when the action is complete.
    */
-  onClick: () => Promise<void>;
+  onClick: () => Promise<unknown>;
```

Comment:
why `unknown` type?

Reply:
one of the click handler now returns something and typescript is not happy about it. i could wrap that one in `() => { doWhatever(); }` but feels this is easier coz this button doesn't care what the handler returns